### ISSUE TITLE
Products Schema against woocommerce

### ISSUE THREAD

**Issue by rollsappletree** [schemapress/Schema#13]
Would be fantastic to have the possibility to add automatic products json-ld schema for woocommerce products!

This would be a killer application!

Cheers!
C

**Comment by renatonascalves** [comment]
WooCommerce already has json-ld schema.

**Comment by hishaman** [comment]
I agree, WooCommercet already has markup, I know it's working fine. However, maybe Schema can expand it at some point in the future.
